### PR TITLE
Complete HTML Forms exercise.

### PR DESCRIPTION
**Pull request by dennis-pnguyen**
Screenshot 2023-07-18 at 11 52 12 AM

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        display: inline;
+      }
+    </style>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="/my-coding-form-page" method="post">
+      <section>
+        <p class="email">Email</p>
+        <label for="email">
```
Comment (shawnkost) [inline html-forms/index.html]: The "email" text should be inside the `label`. The `p` isn't needed

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+          <input
+            type="text"
```
Comment (shawnkost) [inline html-forms/index.html]: This should be `type="email"`

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+            id="email"
+            placeholder="yourname@email.com"
+            name="user_email" />
+        </label>
+      </section>
+
+      <section>
+        <p class="coding">Why is coding cool?</p>
```
Comment (shawnkost) [inline html-forms/index.html]: This should be a `label`

File: html-forms/index.html [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+        <select id="coding" name="codingiscool">
+          <option value="selected">Select an option</option>
+          <option>Because it's interesting</option>
+          <option>Because it's fun</option>
+          <option>Because it's powerful</option>
```
Comment (shawnkost) [inline html-forms/index.html]: These need a `value` attribute

File: html-forms/index.html [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+            id="signup"
+            name="signmeup"
+            value="yes"
+            checked />
+          Sign me up!
+        </label>
+      </section>
+
+      <section>
+        <p class="comment">Comments</p>
```
Comment (shawnkost) [inline html-forms/index.html]: This should be a `label`

**Review by shawnkost — CHANGES_REQUESTED** [review]
Some changes requested here Dennis!

**Comment by dennis-pnguyen** [comment]
Updated!

File: html-forms/index.html [hunk 6]
```diff
@@ -4,72 +4,49 @@
...
-        <p class="coding">Why is coding cool?</p>
-        <select id="coding" name="codingiscool">
-          <option value="selected">Select an option</option>
-          <option>Because it's interesting</option>
-          <option>Because it's fun</option>
-          <option>Because it's powerful</option>
-        </select>
+        <label for="whycoding?">Why is coding cool?</label>
+        <div>
+          <select id="coding" name="codingiscool">
```
Comment (shawnkost) [inline html-forms/index.html]: The `for` needs to either be "coding", or the `id` needs to be "whycoding". The `for `matches the `id`

File: html-forms/index.html [hunk 7]
```diff
@@ -86,13 +63,13 @@ <h1>Coding Form!</h1>
       </section>
 
       <section>
-        <p class="comment">Comments</p>
+        <label for="comments">Comments</label>
         <div>
           <textarea cols="30" rows="8"></textarea>
```
Comment (shawnkost) [inline html-forms/index.html]: This needs an `id` to link to the `label` above ^

**Review by shawnkost — CHANGES_REQUESTED** [review]
Couple changes here Dennis!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**